frontend/src/scenes/plugins/PluginCard.tsx

```diff
@@ -113,7 +130,7 @@ export function PluginCard({ name, description, url, pluginConfig, pluginId, err
                             <Button
                                 type="primary"
                                 loading={loading}
-                                onClick={() => installPlugin(url, 'repository')}
+                                onClick={() => (url ? installPlugin(url, 'repository') : null)}
```

Comment:
We don't actually care about returning a value here, right? If so, the whole function should be ternary'd, i.e. ` onClick={url ? () => { installPlugin(url, 'repository') } : undefined}`